docs/yaml/functions/dependency.yaml

```diff
@@ -180,6 +180,9 @@ kwargs:
       *Since 0.63.0* when the `prefer_static` option is set to `true` the default
       value is `true` otherwise the default value is `false`.
 
+      *WARNING*: the Cuda dependency ignores `prefer_static`, and will treat any value
+      except an explicit value of `true` as `false`.
```

Comment:
Can it be deprecated?

Reply:
I understand. It's really that cuda is meant to be statically linked except in very specific circumstances, so defaulting to static makes sense, but the docs suggest that `prefer_static` will be used.

I *think* this is a documentation issue, but maybe not?

Reply:
Not a behavioral change so it does not have to be addressed now.